dev.budd.Switcheroo.yml

```diff
@@ -0,0 +1,45 @@
+app-id: dev.budd.Switcheroo
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: run.sh
+sdk-extensions:
+  - "org.freedesktop.Sdk.Extension.rust-stable"
+  - "org.freedesktop.Sdk.Extension.llvm16"
+finish-args:
+  - "--share=ipc"
+  - "--socket=fallback-x11"
+  - "--socket=wayland"
+  - "--device=all"
+  - "--filesystem=host-etc"
+modules:
+  - name: switcheroo
+    buildsystem: simple
+    build-commands:
+      - cd switcheroo; cargo build --release --features gui,notify
+      - install -Dm755 run.sh -t /app/bin
+      - install -Dm755 switcheroo/target/release/switcheroo -t /app/bin
+      - install -Dm644 switcheroo/extra/linux/dev.budd.Switcheroo.desktop -t /app/share/applications
+      - install -Dm644 switcheroo/extra/logo/dev.budd.Switcheroo.png -t /app/share/icons/hicolor/256x256/apps
+      - install -Dm644 switcheroo/extra/linux/dev.budd.Switcheroo.metainfo.xml -t /app/share/metainfo
+    sources:
+      - type: git
+        url: https://github.com/budde25/switcheroo.git
+        tag: v0.7.1
+        dest: switcheroo
+      - type: script
+        dest-filename: run.sh
+        commands:
+          - switcheroo gui -vvv "$@"
+    build-options:
+      append-path: "/usr/lib/sdk/rust-stable/bin:/usr/lib/sdk/llvm16/bin"
+      build-args:
+        - --share=network
```

Comment:
This won't work on Flathub, builds are done offline. You have to generate a manifest of cargo dependencies or provide vendored tarballs https://docs.flathub.org/docs/for-app-authors/requirements#dependency-manifest

Reply:
Thanks for the info! I have made the required changes and verified that it builds with the --sandbox flag.